admin/templates/preprint_providers/detail.html

```diff
@@ -39,7 +42,7 @@
                 {% if show_taxonomies %}
```

Comment:
Whats the point of the if statement here?

Reply:
Oh that was just left over, I can remove that.